consensus/parlia/parlia.go

```diff
@@ -882,6 +882,11 @@ func (p *Parlia) assembleVoteAttestation(chain consensus.ChainHeaderReader, head
 			attestation.VoteAddressSet |= 1 << (valInfo.Index - 1) //Index is offset by 1
 		}
 	}
+	validatorsBitSet := bitset.From([]uint64{uint64(attestation.VoteAddressSet)})
+	if validatorsBitSet.Count() < uint(len(signatures)) {
```

Comment:
why not just: 
```
if validatorsBitSet.Count()  < cmath.CeilDiv(len(snap.Validators)*2, 3) {
return fmt.Errorf(...)
}
```
if there are duplicated data in signatures,  bitset count would not equal to len(signatuer), but could still >=2/3 validators size, right?